bokeh/protocol/receiver.py

```diff
@@ -114,11 +116,12 @@ def __init__(self, protocol: Protocol) -> None:
                 fragments.
         '''
         self._protocol = protocol
-        self._current_consumer = self._HEADER
+        self._current_consumer = self._HEADER  # type: ignore[assignment] # https://github.com/python/mypy/issues/2427
```

Comment:
Same as `create`, we would need to make changes in the approach to have better experience with MyPy